src/Compilers/Server/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -1408,5 +1397,64 @@ public void BadKeepAlive4()
                 await serverData.Verify(connections: 2, completed: 2).ConfigureAwait(true);
             }
         }
+
+        [WorkItem(406649, "https://devdiv.visualstudio.com/DevDiv/_workitems?id=406649")]
+        [Fact]
+        public void MissingCompilerAssembly()
```

Comment:
What happened to the mocking test discussion?  I'm still worried about the state of the server when a compiler host throws a `FileNotFoundException`.  Need to ensure we are disconnecting from clients properly by sending the server failed message vs. compilation failed.